.storybook/main.js

```diff
@@ -1,9 +1,6 @@
 module.exports = {
   stories: ['../src/**/*.stories.tsx'],
-  addons: [
-    '@storybook/addon-essentials',
-    'storybook-addon-performance/register',
```

Comment:
Storybook performance doesn't yet support React 18 so I've removed it for now. We can add it back once they do the update